spec/mercure.md

```diff
@@ -271,6 +146,58 @@ eventSource.onmessage = function ({data}) {
 The hub **MAY** require subscribers and publishers to be authenticated, and **MAY** apply extra
 authorization rules not defined in this specification.
 
+# Matcher Types
+
+## Exact Matching
+
+The hub **MUST** support exact matching.
+With this type of matchers, an exact case-sensitive comparison must be made between the topic and the matcher.
+
+The matcher type name is `Exact`.
+The corresponding query parameters are `match` and `matchExact`.
+
+## URL Pattern
+
+The hub **SHOULD** support using URL patterns [@!urlpattern] as matchers.
+URL patterns **SHOULD** be preferred to regular expressions when topics to match are URLs.
+
+The base URL **MUST** be set to the URL of the Mercure hub.
+
+The matcher type name is `URLPattern`.
+The corresponding query parameter is `matchURLPattern`.
+
+## Regular Expression
+
+The hub **SHOULD** support using I-Regexp regular expressions [@!RFC9485] as matchers.
+
+The matcher type name is `Regexp`.
+The corresponding query parameter is `matchRegexp`.
+
+## Common Expression Language (CEL)
+
+The hub **MAY** support using CEL expressions [@cel] as matchers.
+
+A variable named `topics` containing an array of string **MSUT** be passed
+to the expression. This variable **MUST** contain the canonical topic followed by the alternate topics of the update to match.
+
+The hub **MAY** pass other implementation-specific variables and expose implementation-specific functions.
+
+The expression **MUST** return a boolean value: true if the topic matches, false otherwise.
+
+If parsing or checking of a CEL expression fails or if the expression does not return a boolean value, the hub **MUST** return a 400 "Bad Request" HTTP status code.
+
```

Comment:
using topics replaces matcher types? Or you can use both topics and matchers together? Maybe that we're missing some precision about this.

Reply:
I mean, you will have access to a variable named `topics` to use in the CEL expression itself. The matcher type is CEL.

Reply:
The matcher type name is `URITemplate`.
The corresponding query parameter is `matchURITemplate`.